shared/search/user-input/index.stories.js

```diff
@@ -134,68 +135,103 @@ export const makeSelectorMap = (userItems: Array<UserDetails> = maxUsers) => ({
 
 const provider = createPropProvider(makeSelectorMap())
 
+const UserInputEditable = compose(
+  withStateHandlers(props => ({userItems: props.userItems, usernameText: ''}), {
+    onChangeText: () => usernameText => ({usernameText}),
```

Comment:
could this also call the `onChangeText` callback?

Reply:
(same for `oRemoveUser`)

Reply:
Done